src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -67,7 +67,7 @@ internal partial class Interop
     /// Read the GPIO line direction setting.
     /// </summary>
     /// <param name="line">GPIO line handle</param>
-    /// <returns>GPIOD_DIRECTION_INPUT or GPIOD_DIRECTION_OUTPUT.</returns>
+    /// <returns>1 if INPUT otherwise 2 (OUTPUT).</returns>
```

Comment:
This feels like a wrong change; why get rid of the enum/consts?

Reply:
That enum is define and used in C library, doc copied from C library, forgot to update these values